internal/pkg/config/bgp_configs.go

```diff
@@ -1141,7 +1141,7 @@ type ZebraState struct {
 	MplsLabelRangeSize uint32 `mapstructure:"mpls-label-range-size" json:"mpls-label-range-size,omitempty"`
 	// original -> gobgp:software-name
 	// Configure zebra software name.
-	// frr4, cumulus, frr6, frr7, frr7.2, and frr7.3 can be used.
+	// frr4, cumulus, frr6, frr7, frr7.2 and frr7.3 can be used.
```

Comment:
Directly modified?

Reply:
Nope, it appeared with the pyang auto-generation

Reply:
It seems to me there was some manual change of `bgp_configs.go` in b55c32ce65c9b6e0c9f4620e6acb27e0e49f238d
At least, I see that in this commit the comma was present in the file `bgp_configs.go` but not in `tools/pyang_plugins/gobgp.yang`